lib/k8s/client.ex

```diff
@@ -180,7 +180,7 @@ defmodule K8s.Client do
         path_params: [namespace: "test", name: "nginx"]
       }
 
-    Get the nginx deployment in the default namespace:
+    Get the nginx deployment in the default namespace using atom for kind.  e.g. `name: :deployment`:
```

Comment:
The e.g. part is still wrong. How about we remove that as there is a concrete example just below that line?

```suggestion
    Get the nginx deployment in the default namespace by passing the kind as atom.
```